packages/odyssey-react-mui/scripts/properties-to-ts.ts

```diff
@@ -41,55 +42,71 @@ const convert = (baseFiles, propertiesTargetDir) => {
         propertiesTargetDir,
         filename.replace(extension, ".ts")
```

Comment:
do we ever need `filename` WITH the extension?

We can do 

```
 const extension = extname(src);
const filename = basename(src, extension); 
```

that automatically gives you the basename without the extension: https://nodejs.org/api/path.html#pathbasenamepath-suffix

Reply:
I can change this code. Looks like it was existing, but this is a refactor, so I can fix it.

Reply:
4061c0d16cbef1ed4b44506b783705c1ce03544a